.github/workflows/ci.yml

```diff
@@ -181,7 +195,9 @@ jobs:
             -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
             -DIREE_BUILD_COMPILER=OFF \
             -DIREE_RUNTIME_OPTIMIZATION_PROFILE=size \
-            -DIREE_ENABLE_LLD=ON
+            -DIREE_ENABLE_LLD=ON \
+            -DIREE_HAL_DRIVER_CUDA=ON \
+            -DIREE_HAL_DRIVER_HIP=ON 
```

Comment:
We might just want to keep these optional drivers off in the `runtime_small` job. Looking at my other comment, we could use CMake presets for "small" / "minimal" and then drop this standalone job entirely.